dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -69,6 +69,10 @@ implicit.bean.scanning.disabled.CWOWB1009W=CWOWB1009W: Implicit bean archives ar
 implicit.bean.scanning.disabled.CWOWB1009W.explanation=Archives are not scanned for CDI beans unless they have a beans.xml file.
 implicit.bean.scanning.disabled.CWOWB1009W.useraction=Ensure that every archive that uses CDI has a beans.xml file or re-enable implicit beans scanning in your server.xml file.
 
-spi.extension.failed.to.construct.CWOWB1010E=CWOWB1010E: Failed to create CDI Extension with SPI.
-spi.extension.failed.to.construct.CWOWB1010E.explanation=CDI failed to invoke the constructor of the extension class {0} that is registered with the getExtensions() method of a CDIExtensionMetadata implementation because of the exception {1}.
+spi.extension.failed.to.construct.CWOWB1010E=CWOWB1010E: Failed to create CDI Extension {0} with SPI because of the exception {1}.
+spi.extension.failed.to.construct.CWOWB1010E.explanation=CDI failed to invoke the constructor of an extension class that is registered with the getExtensions() method of a CDIExtensionMetadata implementation.
 spi.extension.failed.to.construct.CWOWB1010E.useraction=Fix the error that is preventing your extension from instantiating.
+
+spi.extension.class.in.different.bundle.CWOWB1011E=CWOWB1011E: A class {0} registered through the CDI Extension SPI was in a different bundle to the CDIExtensionMetadata implementation.
+spi.extension.class.in.different.bundle.CWOWB1011E.explanation=The class was registered with the getExtensions() or getBeanClasses method of a CDIExtensionMetadata implementation is in a different bundle to the CDIExtensionMetadata implementation and will be ignored.
```

Comment:
Suggested change to the explanation: "The class is ignored because it was registered with the getExtensions() or getBeanClasses method of a CDIExtensionMetadata implementation and is in a different bundle than the CDIExtensionMetadata implementation."

Reply:
And done. Thank you for the review